apis/eventing/v1alpha1/cloudevent_types.go

```diff
@@ -14,12 +14,18 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package eventemitter
+package v1alpha1
+
+// CloudEventType contains the list of cloudevent types
+// +kubebuilder:validation:Enum=keda.scaledobject.ready.v1;keda.scaledobject.failed.v1
+type CloudEventType string
```

Comment:
I see this validation (and I think that it's useful) but I'm not sure about how this will scale. I mean, if we have 15-20 events, we will have to list all of them here and maybe it doesn't scale. WDYT about adding a check during the admission webhooks?

Reply:
The reason they are here is so that people know what they can use, if we move it then we lose that.

Reply:
I think we can add a check in the admission webhooks later once the number of events exceeds 10